app/controllers/confirmations_controller.rb

```diff
@@ -0,0 +1,10 @@
+class ConfirmationsController < Devise::ConfirmationsController
+
+  private
+
+  def after_confirmation_path_for(resource_name, resource)
+  	byebug
```

Comment:
Plz make it a habbit to remove such code before commiting. Always view diff before committing.
If it is necessary for you to commit such changes temporarily, please add a comment like `#TODO: Remove this` before such code. The `#TODO` tag has special meaning in most of the editors/IDEs